Improve API Docs (#98)
ballerina-platform/module-ballerina-regex: README.md

> This Regex library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.

Author: ```suggestion
The `regex` library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

> This provides functions for regex functionalities like checking whether a string matches a given regex, replacing substrings, and splitting strings based on a regex.

Reviewer: ```suggestion
This provides functions for regex functionalities such as checking whether a string matches a given regex, replacing substrings, and splitting strings based on a regex.
```

This Regex library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.

This provides functions for regex functionalities like checking whether a string matches a given regex, replacing substrings, and splitting strings based on a regex.

> For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).

Reviewer 2: ```suggestion
```

Author: Do we need to remove this from README file too?

Reviewer 2: No. I thought it is module.MD
